cherry_picker/test_cherry_picker.py

```diff
@@ -227,6 +233,27 @@ def test_get_cherry_pick_branch(os_path_exists, config):
     assert cp.get_cherry_pick_branch("3.6") == "backport-22a594a-3.6"
 
 
+@pytest.mark.parametrize("remote_name", ("upstream", "origin", "python"))
+def test_upstream_name(remote_name, config, tmp_git_repo_dir, git_remote):
+    upstream_remote = None
+    if remote_name == "python":
+        upstream_remote = "python"
+    git_remote("add", remote_name, "https://github.com/python/cpython.git")
+    if remote_name != "origin":
+        git_remote("add", "origin", "https://github.com/miss-islington/cpython.git")
+
+    branches = ["3.6"]
```

Comment:
Because it's consistent with the rest of the code in this file. Or more truthfully... Because I copy-pasted one of the other test cases when making this one :)